tests/test_integration.py

```diff
@@ -46,3 +49,11 @@ def mock_get_requests(*args, **kwargs):
 def test_index_page(mock_get_requests, client):
     response = client.get('/')
     assert response.status_code == 200
+
+@mock.patch('requests.get', side_effect=mock_get_requests)
+def test_index_page_contains_card_names(mock_get_requests, client):
+    response_data = client.get('/').data
+    soup = BeautifulSoup(response_data, 'html.parser')
+    all_task_id_and_name_elements = soup.find_all(class_="task-id-and-name")
+    for i in range(3):
+        assert (const.CARD_ARR[i]["name"] in all_task_id_and_name_elements[i].string)
```

Comment:
You could use `enumerate` here:
```suggestion
    for index, card in enumerate(const.CARD_ARR):
        assert (card["name"] in all_task_id_and_name_elements[index].string)
```